### PR TITLE
refactor: remove refresh and replace refresh with render

### PR DESCRIPTION
**Pull request by tdstein**
r:rrarrwr...

### CODE REVIEW

**Comment by tdstein** [comment]
Here are the changes we discussed a little bit ago.

File: src/posit/connect/content.py [hunk 1]
```diff
@@ -171,43 +170,80 @@ def deploy(self) -> tasks.Task:
         ts = tasks.Tasks(self.config, self.session)
         return ts.get(result["task_id"])
 
-    def refresh(self) -> Task | None:
-        """Trigger a content refresh.
+    def rerender(self) -> Task:
```
Comment (nealrichardson) [inline src/posit/connect/content.py:173]: Can this just be `render()`?

**Review by toph-allen — APPROVED** [review]
This looks good to me! Did you decide not to implement different subclasses for now? (I'm fine with that — this looks similar to how the R code currently looks — just curious!)

**Comment by tdstein** [comment]
> This looks good to me! Did you decide not to implement different subclasses for now? (I'm fine with that. This looks similar to how the R code currently looks; I'm just curious!)

Yes, I decided not to for now. I implemented it and realized it wasn't worth the additional maintenance complexity. For what it's worth, I'm comfortable making a breaking change here in the future if we decide to do the abstraction.  At least until we get to a v1.0.0 release.

**Pull request merged**